pkg/onit/env/env.go

```diff
@@ -125,6 +130,9 @@ type ClusterEnv interface {
 	// Topo returns the topo environment
 	Topo() TopoEnv
 
+	// Ran returns the RAN environment
+	Ran() RanEnv
```

Comment:
Since RAN is an acronym, I think according to Go code style guides this should be `RAN()`